TWLight/resources/tests.py

```diff
@@ -695,3 +698,77 @@ def test_csv_formatting(self):
 
         access_codes = AccessCode.objects.all()
         self.assertEqual(access_codes.count(), 0)
+
+
+class AutoWaitlistDisableTest(TestCase):
+    def setUp(self):
+        super(AutoWaitlistDisableTest, self).setUp()
+        editor = EditorFactory()
+        self.user = editor.user
+
+        self.partner = PartnerFactory(
+            status=Partner.WAITLIST,
+            authorization_method=Partner.PROXY,
+            accounts_available=10)
+        self.partner1 = PartnerFactory(
+            status=Partner.WAITLIST,
+            authorization_method=Partner.PROXY,
+            accounts_available=2)
+
+        self.application = ApplicationFactory(
+            editor=editor,
+            status=Application.PENDING,
+            partner=self.partner,
+            rationale='Just because',
+            agreement_with_terms_of_use=True)
+
+        self.application1 = ApplicationFactory(
+            editor=editor,
+            status=Application.PENDING,
+            partner=self.partner1,
+            rationale='Just because',
+            agreement_with_terms_of_use=True)
+
+        self.coordinator = UserFactory(username='coordinator')
+        self.coordinator.set_password('coordinator')
+        coordinators = get_coordinators()
+        coordinators.user_set.add(self.coordinator)
+        self.coordinator.userprofile.terms_of_use = True
+        self.coordinator.userprofile.save()
+
+        for _ in range(2):
```

Comment:
That said, you should double check this, because if it's only creating one authorization I don't see why the test would pass. Perhaps I just misunderstand how this works :)